cylc/flow/time_parser.py

```diff
@@ -355,11 +355,6 @@ def _get_point_from_expression(self, expr, context, is_required=False,
                 if "P" not in item:
                     expr += item
                     continue
-                split_expr = self.OFFSET_REGEX.split(item)
-                expr += split_expr.pop(0)
```

Comment:
This variable is used in the following branch so removing these two lines may have consequences.

Reply:
Argh! Great catch! Fixed.

Reply:
Argh! Bad review prize goes to me :grimacing: